pb/summary/summary.proto

```diff
@@ -110,6 +114,8 @@ message HealthinessInfo {
   // The flakiness out of 100 (think percentage but drop the sign)
   float average_flakiness = 4;
 
+  // The average flakiness for the last interval
+  float previous_flakiness = 5;
```

Comment:
Should this also be repeated?

Reply:
Mm yeah, I agree. If TestInfo's should be repeated then it should be consistent.